remote-finetuning.md

```diff
@@ -0,0 +1,109 @@
+# Fine-Tuning models remotely
+Model fine-tuning in machine learning involves subtly adjusting a pre-existing model, which was initially trained on a larger dataset, to perform a similar but new task using a smaller dataset. If you lack local computing resources, such as a GPU, you can perform this fine-tuning remotely using AI Toolkit and Azure Container App.
+
+## Prerequisites
+1. To run the model fine-tuning in your remote Azure Container App Environment, you need to make sure your subscription have enough GPU capacity amount. Submit a [support ticket](https://azure.microsoft.com/support/create-ticket/) to request the capacity amount required for your application. [Learn More about GPU capacity](https://learn.microsoft.com/en-us/azure/container-apps/workload-profiles-overview)
+2. Make sure you have a [HuggingFace account](https://huggingface.co/) and [generate an access token](https://huggingface.co/docs/hub/security-tokens) if you are using private dataset on HuggingFace or your base model needs access control.
+3. Accept the LICENSE on HuggingFace if you are fine-tuning Mistral or Llama. 
+4. If you enable [Weights & Biases](https://wandb.ai/site), make sure you have a Weights & Biases account and [retrieve your API Key](https://wandb.ai/authorize)
+
+## Provision Azure Resources
+To get started, you need provision the Azure Resource for remote fine-tuning. This can be done by running the `AI Toolkit: Provision Azure Container Apps job for fine-tuning.` from command palette. During this process, you will be prompted to select your Azure Subscription and resource group.
+
+![Provision Fine-Tuning](Images/remote/provision-finetune.png)
+
+You can monitor the progress of the provision via the link that is displayed in the output channel.
+![Provision Progress](Images/remote/provision-progress.png)
+
+## Run
+To start the remote fine-tuning job, execute the `AI Toolkit: Run fine-tuning` command.
+![Run Fine-tuning](Images/remote/run-finetuning.png)
+
+Upon running this command, the extension will do the following operations:
+1. Synchronize your workspace with Azure Files.
+1. Start the ACA job using the commands specified in `./infra/fintuning.config.json`.
+1. Display the job streaming log if the job has started running. 
+    ![Run Log](Images/remote/run-log.png)
+    > **Note:** The job might be queued if there are insufficient resources available. If the log fails to display when the job starts, you can wait for a while and then execute the `AI Toolkit: Show the running fine-tuning job streaming logs` command to re-connect to the streaming log.
+    
+During this process, QLoRA will be used for fine-tuning, and will create LoRA adapters for the model to use during inference.
+The results of the fine-tuning will be stored in the Azure Files.
```

Comment:
Add section for accessing those output files

Reply:
Added this section.